astropy/io/fits/hdu/hdulist.py

```diff
@@ -75,70 +75,53 @@ def fitsopen(name, mode='readonly', memmap=None, save_backup=False,
-        additional optional keyword arguments, possible values are:
-
-        - **uint** : bool
-
-            Interpret signed integer data where ``BZERO`` is the
-            central value and ``BSCALE == 1`` as unsigned integer
-            data.  For example, ``int16`` data with ``BZERO = 32768``
-            and ``BSCALE = 1`` would be treated as ``uint16`` data.
-            This is enabled by default so that the pseudo-unsigned
-            integer convention is assumed.
-
-            Note, for backward compatibility, the kwarg **uint16** may
-            be used instead.  The kwarg was renamed when support was
-            added for integers of any size.
-
-        - **ignore_missing_end** : bool
-
-            Do not issue an exception when opening a file that is
-            missing an ``END`` card in the last header.
-
-        - **checksum** : bool, str
-
-            If `True`, verifies that both ``DATASUM`` and
-            ``CHECKSUM`` card values (when present in the HDU header)
-            match the header and data of all HDU's in the file.  Updates to a
-            file that already has a checksum will preserve and update the
-            existing checksums unless this argument is given a value of
-            'remove', in which case the CHECKSUM and DATASUM values are not
-            checked, and are removed when saving changes to the file.
-
-        - **disable_image_compression** : bool
-
-            If `True`, treats compressed image HDU's like normal
-            binary table HDU's.
-
-        - **do_not_scale_image_data** : bool
-
-            If `True`, image data is not scaled using BSCALE/BZERO values
-            when read.
-
-        - **character_as_bytes** : bool
-
-            Whether to return bytes for string columns. By default this is `False`
-            and (unicode) strings are returned, but this does not respect memory
-            mapping and loads the whole column in memory when accessed.
-
-        - **ignore_blank** : bool
-
-            If `True`, the BLANK keyword is ignored if present.
-
-        - **scale_back** : bool
-
-            If `True`, when saving changes to a file that contained scaled
-            image data, restore the data to the original type and reapply the
-            original BSCALE/BZERO values.  This could lead to loss of accuracy
-            if scaling back to integer values after performing floating point
-            operations on the data.
+    uint : bool
+        Interpret signed integer data where ``BZERO`` is the central value and
+        ``BSCALE == 1`` as unsigned integer data.  For example, ``int16`` data
+        with ``BZERO = 32768`` and ``BSCALE = 1`` would be treated as
+        ``uint16`` data.  This is enabled by default so that the
+        pseudo-unsigned integer convention is assumed.
+
+    ignore_missing_end : bool
+        Do not issue an exception when opening a file that is missing an
+        ``END`` card in the last header.
```

Comment:
We should mention the default here (and for the parameters below). Especially since the defaults are hidden by `kwargs`.